tools/SourceKit/tools/sourcekitd-test/CMakeLists.txt

```diff
@@ -23,9 +23,9 @@ if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     PROPERTIES
     LINK_FLAGS "-Wl,-rpath -Wl,@executable_path/../lib")
```

Comment:
I think if you change this to append to `LINK_FLAGS`, then you won't need to duplicate appending `-fprofile-instr-generate` below -- it will come from AddSwift.cmake.